### PR TITLE
Improve GitHub documentation

### PR DESCRIPTION
**Pull request by cromedome**
To go along with Dancer2 1.x, we should have a more modern and welcoming landing page on GitHub. The core of this is still derived from the Dancer2 Pod, but is written to be prettier and more welcoming. This README is our chance to make a good first impression, and this attempts to make it a much better impression.

Suggestions and feedback are welcome and needed.

### CODE REVIEW

**Review by GeekRuthie — APPROVED** [review]
Love it.  It does read a lot more like, well, a manual, than a POD doc.

File: README.md [hunk 1]
```diff
@@ -1,76 +1,84 @@
...
+  <a href="https://github.com/PerlDancer/Dancer2/discussions">Discussion Forums</a>
+  ·
+  <a href="https://github.com/PerlDancer/Dancer2/wiki">Public Wiki</a>
+  ·
+  <a href="http://lists.preshweb.co.uk/mailman/listinfo/dancer-users">Mailing List</a>
+</p><br>
+
+Dancer2 is the evolution of [Dancer](https://metacpan.org/pod/Dancer)
+which is based on on [Moo](https://metacpan.org/pod/Moo), a lightweight
+object framework for Perl.
```
Comment (yanick) [inline README.md:24]: The original 'which' makes me parse that Dancer1 is based on Moo. 

```suggestion
Dancer2 is the evolution of [Dancer](https://metacpan.org/pod/Dancer)
and is based on on [Moo](https://metacpan.org/pod/Moo), a lightweight
object framework for Perl.
```

**Review by yanick — APPROVED** [review]
Brilliant! Only left one little nitpick, but that's all solid, gorgeous work! :heart:

**Comment by cromedome** [comment]
Thanks for the feedback! Incorporated and merged.

**Pull request closed without merging**